components/MyCollectionList/index.js

```diff
@@ -23,12 +23,14 @@ export default function MyCollectionList() {
     <Flex gap="1.5rem" flexWrap="wrap" alignItems="center" direction="column">
       {starshipsNotForSale.map((starship) => {
         return (
-          <StarshipCollectionCard
-            {...starship}
-            key={starship._id}
-            name={starship.name}
-            img={starship.img}
-          />
+          <>
+            <StarshipCollectionCard
+              {...starship}
+              name={starship.name}
+              img={starship.img}
```

Comment:
Why `name` and `img` in addition  to `{...starship}`, are they not already passed?